### PR TITLE
[Relationship] Enable node connection via some intermediary node

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
**Notes for Reviewers**
1. Supports enabling connection between two entity via third entity, e.g Pod to PV via PVC
2. Added the schema support as well

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: server/meshmodel/schemas/relationship.json [hunk 1]
```diff
@@ -79,8 +94,13 @@
             },
             "description": "Optional fields that are a part of the `to` selector. Absence of a field has an implied * meaning."
           }
+        },
+        "allowMultiple": {
```
Comment (Abhishek-kumar09) [inline server/meshmodel/schemas/relationship.json:98]: This helps us in knowing whether between the same two component multiple connection of same type is available,

Comment (MUzairS15) [reply server/meshmodel/schemas/relationship.json]: After reading the description it seems like there will 2 edges, pod to pvc and then pvc to pv.
Or this schema update is just facilitating the current behaviour we have

Comment (Abhishek-kumar09) [reply server/meshmodel/schemas/relationship.json]: Although we want to have some modification in the code written here. But to your question, it just tells about "via" node, and its on UI implementation to chose any of the two approaches you're mentioning.

**Comment by leecalcote** [comment]
While we're not taking this specific approach, @MUzairS15 @Abhishek-kumar09 this is still an open concern.

**Pull request closed without merging**